package.json

```diff
@@ -15,13 +15,9 @@
   },
   "scripts": {
     "build": "yarn build:keybase-cache && yarn build:ui",
-    "build:testnets": "cd tasks/build/testnets && ./localBuild.sh",
-    "build:gaia": "cd tasks/build/Gaia && ./localBuild.sh",
-    "build:local": "node tasks/build/local/build",
     "build:keybase-cache": "babel-node --presets @babel/env tasks/keybase-fallback.js",
     "build:ui": "cross-env NODE_ENV=production webpack --colors --config webpack.renderer.config.js --mode=production",
-    "start": "concurrently -k \"yarn frontend\" \"yarn backend\"",
-    "start:new": "concurrently -k \"yarn frontend\" \"yarn backend:new\"",
```

Comment:
no more `start:new`?

Reply:
it's always new now, but the account stays always the same